cmd/tracee-ebpf/flags/capabilities.go

```diff
@@ -0,0 +1,105 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/capabilities"
+	"github.com/aquasecurity/tracee/pkg/filters"
+	"kernel.org/pub/linux/libs/security/libcap/cap"
+)
+
+const (
+	CapsMainFlag        = "caps"
+	AllowFailedDropFlag = "allow-failed-drop"
+	CancelDropFlag      = "cancel-drop"
+	AddReqCapsFlag      = "add"
+)
+
+func CapabilitiesHelp() string {
+	return fmt.Sprintf(`Manage the capabilities and capabilities-related operations of tracee-ebpf
+Normally, tracee will drop all capabilities not required to its operations.
+
+Possible options:
+  %-51s allow tracee-ebpf to run with high capabilities, in case that capabilities dropping fails
+  %-51s cancel capabilities drop, so tracee will run with all given capabilities (unless missing required capabilities)
+  %-51s add required capabilities, so tracee won't drop them
+`,
+		AllowFailedDropFlag,
+		CancelDropFlag,
+		fmt.Sprintf("%s=<list_of_caps>", AddReqCapsFlag))
+}
+
+type CapsConfig struct {
+	AllowHighCaps  bool
+	CancelCapsDrop bool
+	CapsToPreserve []cap.Value
+}
+
+func PrepareCapsConfig(capsCfgArray []string) (CapsConfig, error) {
+	var cfg CapsConfig
+	for _, opt := range capsCfgArray {
+		optName := opt
+		var operatorAndValues string
+		operatorIndex := strings.IndexAny(opt, "=!<>")
```

Comment:
Do we care here about any operators besides`=` and `!`? Also, docs don't allow for `!=` so either that logic (and operator) should be removed or it should be documented.